packages/server/src/fhir/operations/ai.ts

```diff
@@ -51,6 +52,14 @@ const operation: OperationDefinition = {
       type: 'string',
       documentation: 'JSON string containing the tools array (optional)',
     },
+    {
+      name: 'stream',
```

Comment:
This is a good idea but maybe we can just rely on the `Accept` header value (of `text/event-stream`) to trigger the streaming behavior?